src/Frontend/Page/Idea.hs

```diff
@@ -169,6 +169,13 @@ linkToIdeaLocation idea = do
              IdeaLocationSpace{} -> "Zum Ideenraum"
              IdeaLocationTopic{} -> "Zum Thema"
 
+-- ASSUMPTION: The caps list does con contain the same value marked as
+-- Clickable and GrayedOut
```

Comment:
i would prefer if this assumption is encoded as an assertion.  (can the test suite then use this assertion to fail in case of error, rather than their independent test function?)